examples/demo_workflow/components/state_redis.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore-actors
+spec:
+  type: state.redis
+  version: v1
+  initTimeout: 1m
+  metadata:
+  - name: redisHost
+    value: localhost:6379
+  - name: redisPassword
+    value: ""
+  - name: actorStateStore
+    value: "true"
```

Comment:
After all that conversation about how redis isn't transactional, is it worth using it for a demo? Seems like vouching for it.

I understand that this is only an example but, just to be sure on the messaging.

Reply:
Actually, leveraging redis store for now as it comes pre-configured with Dapr.
So, the example becomes easy to run and understandable easily.
But, yeah, you have raised a good point.
Let me see if I can get that incorporated in this PR itself.

Reply:
I don't recommend taking this on now. Let's do it after we actually have the redis replacement in place, especially since we're using Redis for all of our other samples and quickstarts.